src/main/java/org/fundacionjala/at15/katas/pokerhands/gonzalo/Compare.java

```diff
@@ -0,0 +1,75 @@
+package org.fundacionjala.at15.katas.pokerhands.gonzalo;
+
+public class Compare {
+    private final int card1 = 0;
+    private final int card2 = 1;
+    private final int card3 = 2;
+    private final int card4 = 3;
+    private final int card5 = 4;
+    private final int difference = 4;
+    private int max;
+    private int min;
+    private Converter valueCard = new Converter();
+    private char[] majorCard;
+    private int[] cardsInt;
+
+    public boolean compareSuit(String[] hand) {
+        boolean verified = false;
+        majorCard = valueCard.converterCharSuit(hand);
+        if (majorCard[card1] == majorCard[card2] && majorCard[card1] == majorCard[card3]
+            && majorCard[card1] == majorCard[card4] && majorCard[card1] == majorCard[card5]) {
+            verified = true;
+        }
+        return verified;
+    }
+
+    public boolean compareCard1(String[] hand) {
```

Comment:
This is good. Try to improve it by avoiding repeating code in `compareCard2`, `compareCard3` and so on. You should be able to have one single method `compareCard` that you could reuse for all cards